contrib/babelfishpg_common/src/collation.c

```diff
@@ -1517,4 +1517,27 @@ babelfish_update_server_collation_name(PG_FUNCTION_ARGS)
 	server_collation_name = pstrdup(babelfish_restored_server_collation_name);
 	MemoryContextSwitchTo(oldContext);
 	PG_RETURN_VOID();
+}
+
+
+/*
+ * print_const_collation - determines whether collation needs to be dumped or not
+ * for collatable data types such as varchar, char, nvarchar, nchar, etc.
+ */
```

Comment:
Let us add more explanation about the case where we do not want to dump. Specifically, const collation id and type collation id.

Reply:
Done